multiplex_immuno_processing/compute_orb_based_registration_paramaters.py

```diff
@@ -0,0 +1,237 @@
+    alignment_offsets_xyz_list = registration_utils.find_xyz_offset_relative_to_ref(
+        img_list, refimg=refimg, ploton=False, verbose=False
+    )
+
+    print(f"alignment offset xyz is {alignment_offsets_xyz_list}")
+
+    #target_img_padded = np.pad(refimg, ((0, 0), (50, 50), (50, 50)), mode='constant')
+    # tifffile.imwrite("refimg.tiff", refimg)
+    alignment_offsets_xyz_list_new_version=[]
+
+    for i in range(len(img_list)):
+        try:
+            print(f"looking at {i}")
+            new_source_img = img_list[i]
+            source_img_FOV_cropped = new_source_img[:, 100:np.shape(new_source_img)[1]-100, 100:np.shape(new_source_img)[2]-100]
+
+            #source_img_padded = np.pad(source_img_FOV, ((0, 0), (50, 50), (50, 50)), mode='constant')
+            print(f"shape of ref img is {np.shape(refimg)}")
+            print(f"shape of source image is {np.shape(source_img_FOV_cropped)}")
+            # tifffile.imwrite(f"source_img_FOV_{i}_padded.tiff", source_img_FOV_cropped)
+
+            #ref_img_cropped = refimg[:,100:np.shape(refimg)[1]-100, 100:np.shape(refimg)[2]-100]
+            # the indexing here is z,y,x -- check this
+            final_y_offset, final_x_offset = registration_utils.perform_alignment(source_img_FOV_cropped, refimg, smaller_fov_modality="source", scale_factor_xy=1, scale_factor_z=1, source_alignment_channel=0, target_alignment_channel=0, source_output_channel=[0], target_output_channel=[0], prealign_z=True, denoise_z=True, use_refinement=False, save_composite=False)
+            print(final_y_offset, final_x_offset)
+            # match_list = align_helper.return_aligned_img_list_new(img_list,offset_list)
+            print("for first img in new_source_img, x_offset is", final_x_offset)
+            print("for first img in new_source_img, y_offset is", final_y_offset)
+            alignment_offsets_xyz_list_new_version.append((final_x_offset, final_y_offset))
+        except:
+            alignment_offsets_xyz_list_new_version.append((0,0))
+
+            print(f"not fitted for {i}")
+    #print("for first img in new_source_img, z_offset is", final_z_offset)
+
+
+    print(f"alignment offset xyz old version is {alignment_offsets_xyz_list}")
+    print(f"alignment offset xyz new version is {alignment_offsets_xyz_list_new_version}")
+
+
+    final_version_offsets_zyx=[]
+    # This just copies over the z offset from the cross correlation method
+    for k in range(len(alignment_offsets_xyz_list_new_version)):
+        dat=alignment_offsets_xyz_list_new_version[k]
+        final_version_offsets_zyx.append([alignment_offsets_xyz_list[k][0], 100-dat[0], 100-dat[1]])
+    print(f"final version offset xyz is {final_version_offsets_zyx}")
+    print("outputing")
+
+
+
+    dfimg["alignment_offsets_xyz"] = final_version_offsets_zyx
+    dfimg["template_position"] = [Position] * dfimg.shape[0]
+    dfout_p = dfimg[["template_position", "align_channel", "alignment_offsets_xyz"]]
```

Comment:
one column here should be the method used to compute the alignment . E.g. `Method` : `Cross corr` or `Method` : `ORB`

Reply:
This is resolved in the new set of commits